src/scss/components/optional/_order-card.scss

```diff
@@ -101,18 +101,20 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
 
     .c-orderCard-order {
         text-align: center;
-        padding: spacing();
+        padding: spacing(x1.5);
         color: $blue;
         font-weight: $font-weight-bold;
         border-top: 1px solid $grey--offWhite;
         line-height: line-height(16px, 24px);
+        @include font-size(base--scaleUp, false);
+        margin-top: spacing();
     }
 
     .c-orderCard-title {
         font-family: $font-family-base;
+        font-weight: 500;
```

Comment:
Should be able to use the font-weight variables for this

Reply:
Changed :+1: